### PR TITLE
IL-39: show default image for ""

### PR DESCRIPTION
**Pull request by hontatfai**
polished the UI with react-native-elements, and added the default image for the empty string.
test
test1

### CODE REVIEW

**Comment by joewetton** [comment]
testers, make sure you do an npm install before running :)

File: app/components/inbox/index.js [hunk 1]
```diff
@@ -28,47 +19,52 @@ class Inbox extends Component {
...
             </View>
         );
     }
 
     renderItem = ({item, index}) => {
         /* get author name and portrait for each message */
-        let author = '';
-        let portrait = '';
+        let author = null;
+        let portrait = 'https://www.mautic.org/media/images/default_avatar.png';
```
Comment (joewetton) [inline app/components/inbox/index.js]: Download this png image and store it locally inside our app eg. '../assets/default_avatar.png'

**Review by joewetton — CHANGES_REQUESTED** [review]
The polishes to the UI you made look really good! Also can confirm the default silhouette character image displays for me. Just one small change other than that IOS is good to go 👍
img_1117

**Comment by hontatfai** [comment]
@joewetton I tried to store the local image into a variable before, but it doesn't work;  Recact is weird, it seems only work on the url. I only can upload the image to our github, and use our url to import it.
Some methods I tried, but doesn't work:
1. let portrait = new Image();
    portrait = require ('../../assets/*.png');
2. let portrait = '../../assets/*.png';  //as you suggested 
3. const d_image = {
        img: require ('../../assets/*.png')
    };
    let portrait = d_image.img;

Oh, and remember when we merge request, need to change the  url too.
Now it is under the branch IL-39. 
https://raw.githubusercontent.com/bnelson777/id.ly/IL-39/app/assets/default_avatar.png
 I am worry, we will delete the branches. When we merged, its ulr probably will changed to
https://raw.githubusercontent.com/bnelson777/id.ly/master/app/assets/default_avatar.png

**Comment by hontatfai** [comment]
@joewetton my bad :sweat_smile:  I found the issue why I got the red screen yet, now, it works.

**Review by joewetton — APPROVED** [review]

**Review by Vancev — APPROVED** [review]
Looks good on android

**Pull request merged**